api/extension-guides/web-extensions.md

```diff
@@ -315,7 +315,11 @@ npx: installed 78 in 2.196s
 
 ```
 
-Then, in another terminal, run `npx localtunnel -p 5000`:
+If you are using a Chromium based browser, open vscode.dev, run Developer: Install Web Extension... from the Command Palette and give "http://localhost:5000" as the address.
+
+<details><summary>If you are not using a Chromium browser</summary>
```

Comment:
Suggestion: write out `If you are not using Edge, Chrome, nor any other Chromium browser:`